pkgs/applications/audio/distrho/default.nix

```diff
@@ -71,6 +89,7 @@ stdenv.mkDerivation rec {
         tal-vocoder-2
         temper
         vex
+        vitalium
         wolpertinger
     '';
     license = with licenses; [ gpl2 gpl3 gpl2Plus lgpl3 mit ];
```

Comment:
Can you please update them to use Only and Plus variants.

Reply:
Done. The various plugins are licensed differently (and some unclearly), so it's quite a list.